apps/console/src/features/core/constants/app-constants.ts

```diff
@@ -31,16 +33,103 @@ export class AppConstants {
     private constructor() { }
 
     /**
-     * Base path for Admin features.
-     * @type {string}
+     * Get the admin view base path.
+     *
+     * @return {string}
      */
-    public static readonly ADMIN_VIEW_BASE_PATH: string = window["AppUtils"].getConfig().adminApp.basePath;
+    public static getAdminViewBasePath(): string {
+        return window["AppUtils"].getConfig().adminApp.basePath;
+    }
 
     /**
-     * Base path for Developer features.
-     * @type {string}
+     * Get the admin view base path.
+     *
+     * @return {string}
+     */
+    public static getDeveloperViewBasePath(): string {
```

Comment:
Shouldn't we move this to utils?

Reply:
That makes more sense. Let's move these to a util in a new PR.